java-speech/google-cloud-speech/src/main/java/com/google/cloud/speech/v1/stub/HttpJsonSpeechStub.java

```diff
@@ -139,14 +151,154 @@ public class HttpJsonSpeechStub extends SpeechStub {
+                      .setPath(
+                          "/v1/operations",
+                          request -> {
+                            Map<String, String> fields = new HashMap<>();
+                            ProtoRestSerializer<ListOperationsRequest> serializer =
+                                ProtoRestSerializer.create();
+                            return fields;
+                          })
+                      .setQueryParamsExtractor(
+                          request -> {
+                            Map<String, List<String>> fields = new HashMap<>();
+                            ProtoRestSerializer<ListOperationsRequest> serializer =
+                                ProtoRestSerializer.create();
+                            serializer.putQueryParam(fields, "$alt", "json;enum-encoding=int");
+                            return fields;
+                          })
+                      .setRequestBodyExtractor(request -> null)
+                      .build())
+              .setResponseParser(
+                  ProtoMessageResponseParser.<ListOperationsResponse>newBuilder()
+                      .setDefaultInstance(ListOperationsResponse.getDefaultInstance())
+                      .setDefaultTypeRegistry(typeRegistry)
+                      .build())
+              .build();
+
+  private static final ApiMethodDescriptor<GetOperationRequest, Operation>
+      getOperationMethodDescriptor =
+          ApiMethodDescriptor.<GetOperationRequest, Operation>newBuilder()
+              .setFullMethodName("google.longrunning.Operations/GetOperation")
+              .setHttpMethod(HttpMethods.GET)
+              .setRequestFormatter(
+                  ProtoMessageRequestFormatter.<GetOperationRequest>newBuilder()
+                      .setPath(
+                          "/v1/operations/{name=**}",
+                          request -> {
+                            Map<String, String> fields = new HashMap<>();
+                            ProtoRestSerializer<GetOperationRequest> serializer =
+                                ProtoRestSerializer.create();
+                            serializer.putPathParam(fields, "name", request.getName());
+                            return fields;
+                          })
+                      .setQueryParamsExtractor(request -> new HashMap<>())
+                      .setRequestBodyExtractor(request -> null)
+                      .build())
+              .setResponseParser(
+                  ProtoMessageResponseParser.<Operation>newBuilder()
+                      .setDefaultInstance(Operation.getDefaultInstance())
+                      .build())
+              .setOperationSnapshotFactory(
+                  (request, response) -> HttpJsonOperationSnapshot.create(response))
+              .setPollingRequestFactory(
+                  compoundOperationId ->
+                      GetOperationRequest.newBuilder().setName(compoundOperationId).build())
```

Comment:
This had to be added as in manually. Didn't work with the generator (as of now).

Reply:
I create a new version of Gax-HttpJson with an updated HttpJsonLongRunningClient. This version of the LongRunningClient doesn't need the OperationSnapshotFactory and PollingRequestFactory: https://github.com/lqiu96/gapic-generator-java/pull/1

The implementation was taken from the GrpcLongRunningClient.